Makefile

```diff
@@ -3,6 +3,9 @@ EPOCH_TEST_COMMIT ?= f2925f58acc259c4b894353f5fc404bdeb40028e
 PROJECT := github.com/kubernetes-incubator/cri-containerd
 BINDIR ?= ${DESTDIR}/usr/local/bin
 BUILD_DIR ?= _output
+# VERSION is the version of the binary.
+# TODO: push a tag so this will not print a fatal error
+VERSION:=$(shell git describe --tags --dirty)
```

Comment:
I sort of like getting the version from a flat file rather than a git tag and the commit from git... see opencontainers/runc for a easy to use pattern.  @Random-Liu wdyt?